actions/sharepoint.js

```diff
@@ -385,9 +385,14 @@ async function deleteFloodgateDir() {
     return deleteSuccess;
 }
 
-async function updateExcelTable(excelPath, tableName, values) {
+async function updateExcelTable(excelPath, tableName, values, isGraybox = false) {
     const { sp } = await getConfig();
-    const itemId = await getItemId(sp.api.file.get.baseURI, excelPath);
+    let itemId = '';
+    if (isGraybox) {
```

Comment:
Ideally we want to avoid doing this in sharepoint.js. But this was a small enough change. So I did not want to create a copy of the whole function just for this.

Reply:
Are we going to use the same fgBaseURI property for grayBox?

Reply:
Yes, although the root folder will be the graybox root rolder which is read in appConfig.js as done in this PR.

The uri is generated as `sharepoint site uri` + `root-folder`. The `gbRootFolder` value gets applied to the sharepoint uri to generate the fgBaseUri. 

This is to re-use the sharepoint auth, config and other classes/setup as-is as much as possible. 

This will give us a lot of things for free for graybox and dont have to setup everything from scratch.